src/js/tracker.js

```diff
@@ -202,6 +201,12 @@
 			// Default is user agent defined.
 			configCookiePath = '/',
 
+			// First-party cookie samesite attribute
+			configCookieSameSite = argmap.hasOwnProperty('cookieSameSite') ? argmap.cookieSameSite : 'None',
+
+			// First-party cookie secure attribute
+			configCookieSecure = argmap.hasOwnProperty('cookieSecure') ? argmap.cookieSecure : true,
```

Comment:
Instead of creating a new config setting here, could you test if `window.location.protocol == 'http'`?

Reply:
I could but a user might wish to set `Lax` and not Secure cookies even on HTTPS sites, so that the same cookie is accessible on HTTP too. 
I do quite like the idea of using `window.location.protocol == 'https'` as the default option instead of just `true`. My worry is with Chromes update SameSite=None and not secure are an incompatible set of default values (although SameSite=None cookies don't work on http at all with the latest changes as they have to be Secure). So with that in mind, these cookies are just not going to work well on HTTP sites anymore so I've come to the conclusion that extra code to manage http sites just doesn't add much value. If users want to use the tracker on http then they need to toggle this to `false` and `Lax`.

Reply:
Your answer is very reasonable.  My suggestion was trying to protect http users from having to make a config change.  But actually it makes sense that http users should explicitly control each parameter.